<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HighScorePage</class>
 <widget class="QWidget" name="HighScorePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="lblHighScoreTitle">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/higscores.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblHighscore3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHighscore1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHighscore10">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHighscore5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHighScoreNumbers_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>81</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>6.

7.

8.

9.

10.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblHighscore2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHighscore7">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHighScoreNumbers">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>81</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>1.

2.

3.

4.

5.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblHighscore9">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHighscore6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHighscore8">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>180</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHighscore4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReturnHome">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>140</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName9">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName10">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNewHighScoreInfo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>581</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Your scored placed in the top 10 of all time!!!! 
Enter your name to be remembered for 
                     all eternity!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledNewName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEnterName">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Name</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
